source/brailleTables.py

```diff
@@ -76,7 +76,7 @@ def listTables():
 	"mn-MN.utb":"mn-MN-g1.utb",
 	"nl-BE-g1.ctb":"nl-BE-g0.utb",
 	"nl-NL-g1.ctb":"nl-NL-g0.utb",
-	"no-no.ctb":"no-no-comp8.ctb",
+	"no-no.ctb":"no-no-8dot.utb",

```

Comment:
Looks like I didn't express myself clear enough.

You need to cover both no-no.ctb and no-no-comp8.ctb renames.

```suggestion
	"no-no.ctb":"no-no-8dot.utb",
	"no-no-comp8.ctb":"no-no-8dot.utb",
```